locksmith/src/lock.js

```diff
@@ -2,3 +2,4 @@ const models = require('../models')
 
 models.Lock.removeAttribute('id')
 module.exports = models.Lock
+export default models.Lock
```

Comment:
For the moment I kept both kinds of exports on these, the line addition is just so I could import as an ES6 module from TS

Reply:
I think this created conflicts :( sorry.